src/Datadog.Trace.ClrProfiler.Managed/Integrations/WcfIntegration.cs

```diff
@@ -163,6 +163,7 @@ private static Scope CreateScope(object requestContext)
                     tags,
                     tagsFromHeaders);
 
+                tags.Measured = 1;
```

Comment:
Looks like we already add the metric in `span.DecorateWebServerSpan(...)` above, so we don't need to add it again here.